llvm/lib/Target/AMDGPU/GCNRegPressure.cpp

```diff
@@ -261,15 +257,11 @@ void GCNRPTracker::reset(const MachineInstr &MI,
   MaxPressure = CurPressure = getRegPressure(*MRI, LiveRegs);
 }
 
-void GCNUpwardRPTracker::reset(const MachineInstr &MI,
-                               const LiveRegSet *LiveRegsCopy) {
-  GCNRPTracker::reset(MI, LiveRegsCopy, true);
-}
-
 void GCNUpwardRPTracker::reset(const MachineRegisterInfo &MRI_,
                                const LiveRegSet &LiveRegs_) {
   MRI = &MRI_;
   LiveRegs = LiveRegs_;
+  LastTrackedMI = nullptr; // TODO: LastTrackedMI isnt' used, remove?
```

Comment:
Just remove it

Reply:
I haven't noticed it's used with the base class accessor, so I decided to leave it and delete TODO comment.